api/web3server_utils.go

```diff
@@ -24,7 +24,7 @@ import (
 	"google.golang.org/grpc/status"
 
 	logfilter "github.com/iotexproject/iotex-core/api/logfilter"
-	"github.com/iotexproject/iotex-core/ioctl/util"
+	ioctl "github.com/iotexproject/iotex-core/ioctl/util"
```

Comment:
`ioctlUtil` is preferred

Reply:
@Liuhaai we shouldn't use ioctl/util. You may move this function into go-pkgs/util or somewhere to be shared by ioctl and api. IMO, even duplication of the code is better than refer to ioctl/util

Reply:
Per @dustinxie's comment, I'll just duplicate the code. Perhaps in a future diff, I could contribute to iotexproject/go-pkgs/util and add IoAddrToEvmAddr there (depending on what happens with this diff, you can see below I have some questions about importing ioctl/output).

Reply:
I filed [a new pr](https://github.com/iotexproject/iotex-core/issues/2934) which is to move the func to `go-pkg` package. Feel free to contribute that pr if you want